components/automate-cli/cmd/chef-automate/cmdUtils.go

```diff
@@ -382,7 +388,7 @@ func appendChildFileToParentFile(hostIp, parent, child string) (string, error) {
 
 // checkResultOutputForError checks If the output contains the word "error" then return error
 func checkResultOutputForError(output string) error {
-	if strings.Contains(strings.ToUpper(strings.TrimSpace(output)), "ERROR") {
+	if strings.Contains(strings.ToUpper(strings.TrimSpace(output)), "ERROR") && !strings.Contains(strings.ToUpper(strings.TrimSpace(output)), strings.ToUpper("PreflightError: One or more preflight checks failed")){
```

Comment:
for long expressions break into multiple lines for better readability

Reply:
Done